common/lib/xmodule/xmodule/modulestore/django.py

```diff
@@ -243,9 +244,38 @@ class ModuleI18nService(object):
     i18n service.
 
     """
-
-    def __getattr__(self, name):
-        return getattr(django.utils.translation, name)
```

Comment:
We should document the methods we support on the i18n service in the XBlock docs.